aws-inventory/cloudformation/Inventory-Template.yaml

```diff
@@ -829,6 +829,48 @@ Resources:
           Type: SNS
           Properties:
             Topic: !Ref TriggerAccountInventoryFunctionTopic
+  
+  StorageGatewayInventoryLambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub "${pResourcePrefix}-storage-gateway-inventory"
```

Comment:
One thing missing - down in the dashboard, there are entries for each function created that show # of invocations, # errors and a max duration graph. These three functions should be added there.